.clang-format

```diff
@@ -53,14 +53,15 @@ IncludeCategories:
   - Regex: '.*'
     Priority: 3
 IndentCaseLabels: true
-IndentWidth: 2
+IndentWidth: 4
 IndentWrappedFunctionNames: false
+InsertNewlineAtEOF: true
```

Comment:
This is the reason why I went through the trouble of installing a newer clang-format version (16) than what comes with Ubuntu (I think 12? Can't remember). Lol